internal/service/privatelinkendpointserviceserverless/resource_privatelink_endpoint_service_serverless.go

```diff
@@ -146,6 +148,51 @@ func resourceCreate(ctx context.Context, d *schema.ResourceData, meta any) diag.
 	return resourceRead(ctx, d, meta)
 }
 
+func resourceUpdate(ctx context.Context, d *schema.ResourceData, meta any) diag.Diagnostics {
+	connV2 := meta.(*config.MongoDBClient).AtlasV2
+	projectID := d.Get("project_id").(string)
+	instanceName := d.Get("instance_name").(string)
+	endpointID := d.Get("endpoint_id").(string)
+
+	_, _, err := connV2.ServerlessPrivateEndpointsApi.GetServerlessPrivateEndpoint(ctx, projectID, instanceName, endpointID).Execute()
+	if err != nil {
+		return diag.Errorf("error getting Serverless PrivateLink Endpoint Information: %s", err)
+	}
+
+	// only "comment" attribute update is supported, updating other attributes forces replacement of this resource
```

Comment:
we might want to do in the beginning of update function something like:

if ! d.HasChange("comment") {
	return resourceRead(ctx, d, meta)	
	}

Reply:
added